packages/edit-site/src/components/page-pages/index.js

```diff
@@ -464,9 +468,27 @@ export default function PagePages() {
 		[ authors, view.type, frontPageId, postsPageId ]
 	);
 
+	const onActionPerformed = useCallback(
+		( actionId, items ) => {
+			if (
+				actionId === 'move-to-trash' ||
+				actionId === 'permanently-delete'
+			) {
+				setSelection(
+					selection.filter(
+						( id ) =>
+							! items.some( ( item ) => getItemId( item ) === id )
```

Comment:
Why not set it to []?

Reply:
I may have multiple items selected some could be deleted some could not. The delete may only apply to some items I should not clear the others. For example on pages I may select a filter on status for trash and drafts, I select a trashed item and draft I use the delete action. The items that was already in the trash should not be unselected nothing changed on that item.